flink-dist/src/main/flink-bin/bin/flink-daemon.sh

```diff
@@ -85,6 +85,29 @@ id=$([ -f "$pid" ] && echo $(wc -l < "$pid") || echo "0")
 FLINK_LOG_PREFIX="${FLINK_LOG_DIR}/flink-${FLINK_IDENT_STRING}-${DAEMON}-${id}-${HOSTNAME}"
 log="${FLINK_LOG_PREFIX}.log"
 out="${FLINK_LOG_PREFIX}.out"
+gclog="${FLINK_LOG_PREFIX}.gc_log"
+
+if [ ${FLINK_JVM_GC_LOGGING} ];then
+    FLINK_JVM_GC_LOGGING_OPTS="-XLoggc:${gclog} " \
+      "-XX:+PrintGCApplicationStoppedTime " \
```

Comment:
do we need to print the value of `-XX:+PrintGCApplicationConcurrentTime`

Reply:
It' ok to add this option, but in my experience, we usually do not care about the ran time between each stop. So I prefer to not add it, but it's ok to add it if it's needed and useful.